Gems/LmbrCentral/Code/Source/Shape/EditorSphereShapeComponent.cpp

```diff
@@ -120,4 +157,34 @@ namespace LmbrCentral
             }
         }
     }
+
+    float EditorSphereShapeComponent::GetRadius() const
+    {
+        return m_sphereShape.GetSphereConfiguration().m_radius;
+    }
+
+    void EditorSphereShapeComponent::SetRadius(float radius)
+    {
+        m_sphereShape.SetRadius(radius);
+    }
+
+    AZ::Vector3 EditorSphereShapeComponent::GetTranslationOffset() const
+    {
+        return m_sphereShape.GetTranslationOffset();
+    }
+
+    void EditorSphereShapeComponent::SetTranslationOffset(const AZ::Vector3& translationOffset)
+    {
+        m_sphereShape.SetTranslationOffset(translationOffset);
+    }
+
+    AZ::Transform EditorSphereShapeComponent::GetManipulatorSpace() const
+    {
+        return GetWorldTM();
+    }
+
+    AZ::Quaternion EditorSphereShapeComponent::GetRotationOffset() const
+    {
+        return AZ::Quaternion::CreateIdentity();
+    }
```

Comment:
question: I might have missed it but these all seem to return the identity, is the plan in future to move to support rotation offset?

Reply:
Cool thanks for clarifying